UnityProject/Assets/Scripts/Systems/Antagonists/AntagManager.cs

```diff
@@ -193,9 +193,9 @@ public void RemindAntags()
 		/// </summary>
 		public void ShowAntagStatusReport()
 		{
-			StringBuilder statusSB = new StringBuilder($"<color=white><size=60><b>End of Round Report</b></size></color>\n\n", 200);
+			StringBuilder statusSB = new StringBuilder();
 
-			var message = $"End of Round Report on {ServerData.ServerConfig.ServerName}\n";
+			var message = $"";
```

Comment:
Are these changes desirable? You're just PRing station objectives, right?

Reply:
where did that change even come from??? i didn't do that

Reply:
OH WAIT I REMEMBER NOW
I moved that part to the station objectives script, so that it would put the station objectives first and then the antag objectives.